src/signature.cpp

```diff
@@ -104,7 +104,7 @@ struct MappedMem
  *
  * @return RSA Object
  */
-RSA* createPublicRSA(const std::string& publicKey)
+EVP_PKEY_Ptr createPublicRSA(const std::string& publicKey)
```

Comment:
I guess the current change will bring new logic to managing public keys from explicit RSA to generic EVP PublicKEY. Thun, that function could be renamed to more generic naming, e,g, `createPublicKey`, and the next places where it is used.
The @doc should also be actualized.

Reply:
fixed